README.md

```diff
@@ -1,15 +1,17 @@
-# Ruby Lecture: Introduction - Reading Error Messages
+# Intro to Reading Ruby Error Messages
 
+## Overview
 
+In this lab, you'll be reading error messages from tests. This lab is designed so that both running the files *and* running the test suite via the `learn` command will show the error messages for you to decode. Moving forward though, you'll be reading error messages mainly through running the test suite. 
 
 ## Objectives
 
-1. Understand how to read error messages.
-2. Practice fixing some common errors.
-3. Practice using the `learn` command in your terminal to run the tests for a lab.
-4. Practice submitting a lab. 
+1. Read the three different parts of an errors message.
+2. Identify four error types- name errors, syntax errors, types errors, and division errors- and fix them
```

Comment:
should we also add argument errors ? Don't know how relevant it is for this lecture